test/src/org/exist/xupdate/modifications/append_child.xml

```diff
@@ -5,7 +5,7 @@
 		<xu:element name="email"><xu:attribute name="type">home</xu:attribute>andrew@foo.org</xu:element>
 	</xu:append>
 	
-	<xu:append select="//address[lname = 'Jekyll']" child="3">
+	<xu:append select="//address[lname = 'Jekyll']" child="5">
```

Comment:
Well, like I say the test result fragment expect it to be element 5. So either we change the update here or the result fragment. But if we want the test to fail and still have travis to stop on any failure ...